lib/daos/ridesDao.js

```diff
@@ -0,0 +1,50 @@
+import { badImplementation } from '@utils/responseInterceptors';
+import moment from 'moment';
+import { TRIP_STATUS_TYPES } from '@utils/constants';
+import { models } from '@models';
+
+export const bookARide = async (pickUpLocation, dropLocation, custId) => {
+  try {
+    const ridesData = {
+      pickupLoc:pickUpLocation,
+      dropLoc: dropLocation,
+      dateOfRide: moment(moment.now()).format('YYYY-MM-DD'),
+      bookingTime: moment(moment.now()).format("YYYY-MM-DD HH:mm:ss"),
+      custId,
+      created_at: moment(moment.now()).format("YYYY-MM-DD HH:mm:ss")
+    }
+    return await models.requested_rides.create(ridesData);
```

Comment:
```suggestion
    return await models.requestedRides.create(ridesData);
```
Let's keep model name camelcase